web_timeline/static/lib/vis/vis-timeline-graph2d.min.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
@Creamaster PLZ We must keep the minified version of the JS and CSS files to ease the debug. All the js and css files are automatically minified by Odoo when the application run in normal mode. But once you activate the debug mode, you get the original file and it's very useful to ease the debug.

Reply:
That was my intent, but the non-minified version isn't distributed anymore : 
https://github.com/almende/vis/tree/master/dist

Reply:
@Creamaster Ok